### ISSUE TITLE
Load reviews via Ajax

### ISSUE THREAD

**Issue by kevinwhoffman** [WordImpress/WP-Business-Reviews#63]
## Issue Overview

Since #12 was resolved, review sources are now populated via Ajax. The `Get Reviews` buttons need connected to PHP methods via Ajax that will populate the reviews.

## Tasks

- [x] Attach event listeners to `Get Reviews` buttons once review sources are populated.
- [x] Use click event to trigger Ajax request.
- [x] Return normalized review data.
- [x] Update reviews in Builder preview.
